rpc/src/v1/impls/web3.rs

```diff
@@ -34,4 +36,15 @@ impl Web3 for Web3Client {
 			_ => Err(Error::invalid_params())
 		}
 	}
+
+	fn sha3(&self, params: Params) -> Result<Value, Error> {
+		from_params::<(Bytes,)>(params).and_then(
+			|(data,)| {
+				let sha3 = data.to_vec().sha3();
```

Comment:
it's a byte array as i guess all the `data` specified in the json-rpc
there is only one example in wiki and it's a hex-encoded bytes